src/WebClient.spec.js

```diff
@@ -191,6 +191,19 @@ describe('WebClient', function () {
       });
     });
 
+    it('should return platform error if 200 request is not ok', function () {
```

Comment:
can we rephrase the description of this test case a bit? how about `should fail with platform errors when the API response is an error`?

Reply:
rephrased